source/administration/config-settings.rst

```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
 | This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Global SMTP Server Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.
```

Comment:
Actually the default value for this configuration parameter is `1800` seconds, and it shouldn't be `0`. Also, there is no maximum value for this.

Reply:
Great! Is "milliseconds" in the first sentence correct?

Reply:
Mmm, It should be `seconds` as well.